packages/gatsby-cli/src/reporter/loggers/ink/components/error.tsx

```diff
@@ -0,0 +1,99 @@
+import React from "react"
+import path from "path"
+import { Color, Box } from "ink"
+import { get } from "lodash"
+
+interface IFileProps {
+  filePath: string
+  location: string
+}
+const File = ({ filePath, location }: IFileProps): JSX.Element => {
+  const lineNumber = get(location, `start.line`)
+
+  let locString = ``
+  if (typeof lineNumber !== `undefined`) {
+    locString += `:${lineNumber}`
+    const columnNumber = get(location, `start.column`)
+    if (typeof columnNumber !== `undefined`) {
+      locString += `:${columnNumber}`
+    }
+  }
+
+  return (
+    <Color blue>
+      {path.relative(process.cwd(), filePath)}
+      {locString}
+    </Color>
+  )
+}
+
+interface IDocsLinkProps {
+  docsUrl: string
+}
+const DocsLink = ({ docsUrl }: IDocsLinkProps): JSX.Element => {
+  // TODO: when there's no specific docsUrl, add helpful message describing how
+  // to submit an issue
+  if (docsUrl === `https://gatsby.dev/issue-how-to`) return null
```

Comment:
I think the return type of DocsLink needs to be `React.ReactNode` in order to support null